pom.xml

```diff
@@ -84,6 +84,24 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<!--H2 dependency-->
+		<dependency>
+			<groupId>com.h2database</groupId>
+			<artifactId>h2</artifactId>
+			<version>1.4.200</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
```

Comment:
h2database dependency is only required for test profile so only one dependency with <scope>test<.scope> is needed, remove this one.